script/cmake-build-android

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
```

Comment:
can we extend `script/cmake-build` to support android platform?

Reply:
I added the changes to `script/cmake-build`